hack/start_db.sh

```diff
@@ -8,4 +8,5 @@ mkdir -p /tmp/postgres/data
 mkdir -p /tmp/postgres/sockets
 
 initdb -D /tmp/postgres/data -U postgres
+
```

Comment:
Not much going on in this file anymore, I suspect you can just remove this newline to omit it from the commit

Reply:
That is completely right, I will.

Reply:
Done.